Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -762,7 +763,7 @@ function vtkRenderWindowInteractor(publicAPI, model) {
       if (pointers.length === 0) {
         const callData = {
           ...getModifierKeysFor(EMPTY_MOUSE_EVENT),
-          position: pointers[0].position,
+          position: undefined,
```

Comment:
This should be `getScreenEventPositionFor(event)`. Thanks for catching this issue!